app/profile/SkillsTab.tsx

```diff
@@ -11,7 +15,14 @@ import skills from "~/src/skills/skills.json";
 export const SkillsTab = () => {
   const { model, updating, setModel } = usePatchedJSONResource<UserProfile>({
     url: "/profile/me",
-    writeKeys: ["skills"],
+    writeKeys: ["skills", "bio"],
+  });
+
+  const actualBio = model?.bio ?? "";
+
+  const [bioState, setBioState] = useState({
```

Comment:
Jako čtenáři by mně tady pomohlo, kdyby ten typ byl definovaný. Takhle si ho TypeScript odvodí, ale odvodí si ho moc volně:

```typescript
const bioState: {
    bio: string;
    submissionState: string;
}
```

Je lepší, když je ten `submissionState` rovnou union (`"changes_done" | "submitted_successfully" | …`), líp to pak doplňuje a chytá případné chyby.